tests/dmctl_basic/conf/dm-master.toml

```diff
@@ -1,4 +1,5 @@
 # Master Configuration.
 
+name = "master1"
```

Comment:
if user didn't set this field, could `get-config` find this config? if so, should we set a default value

Reply:
We adjust master name for user. 
https://github.com/pingcap/dm/blob/master/dm/master/config.go#L250-L257
They may use list-member to check it.